features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/domain/event/CoreTracerSpanToSpanEventMapper.kt

```diff
@@ -103,10 +96,12 @@ internal class CoreTracerSpanToSpanEventMapper(
             session = event.tags[LogAttributes.RUM_SESSION_ID]?.let { SpanEvent.Session(it as? String) },
             view = event.tags[LogAttributes.RUM_VIEW_ID]?.let { SpanEvent.View(it as? String) }
         )
+        val mostSignificantTraceId = event.traceId.toHighOrderLong().toHexString()
```

Comment:
actually this naming `toHighOrderLong` is more correct imo than using `mostSignificantTraceId` / `leastSignificantTraceId` (used in multiple places for variables/methods), because latter naming is usually representing single bit, while we are taking higher/lower chunk. Although `mostSignificant64Bits` also work, because it explicitly says how much do we take.

Reply:
going to change those methods in the BigIntegerUtils